examples/platform/silabs/SiWx917/matter_config.cpp

```diff
@@ -154,47 +105,31 @@ CHIP_ERROR EFR32MatterConfig::InitMatter(const char * appName)
     SILABS_LOG("Init CHIP Stack");
     // Init Chip memory management before the stack
     ReturnErrorOnFailure(chip::Platform::MemoryInit());
+
+    SILABS_LOG("Init RSI 911x Platform");
+    int a = wfx_rsi_platform();
```

Comment:
this` int a` is never used also use a more explicit name

Reply:
updated in latest commit